samples/wavlib/src/main/cpp/player/SimpleMultiPlayer.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// wavlib includes
+#include <io/stream/MemInputStream.h>
+#include <io/wav/WavStreamReader.h>
+#include <player/OneShotSampleBuffer.h>
+
+#include "SimpleMultiPlayer.h"
+
+static const char* TAG = "SimpleMultiPlayer";
+
+using namespace wavlib;
+
+constexpr int kBufferSizeInBursts = 2; // Use 2 bursts as the buffer size (double buffer)
+
+SimpleMultiPlayer::SimpleMultiPlayer(int numSampleBuffers, int channelCount, int sampleRate) {
+    mSampleBuffers = new OneShotSampleBuffer[mNumSampleBuffers = numSampleBuffers];
+    openStream(channelCount, sampleRate);
+}
+
+DataCallbackResult SimpleMultiPlayer::onAudioReady(AudioStream *oboeStream, void *audioData,
+        int32_t numFrames) {
+
+    memset(audioData, 0, numFrames * sizeof(float));
```

Comment:
* channelCount

Reply:
This is all implicitly mono at this point. I'd like to plumb this correctly when I do the stereo/mixable version.